kod/object/active/holder/nomoveon/battler/monster/towns/Goad.kod

```diff
@@ -1208,6 +1225,16 @@ messages:
               return FALSE;
            }
 
+      %%% Summoned Monsters just seem to fall into the lava.
+      %%% For now Goad disallows their use - November 2023
```

Comment:
"For now" is likely to be years or decades.. strengthen comment.

Reply:
Ok - done.